docs/Running_unit_tests.md

```diff
@@ -49,3 +49,16 @@ Our components written in Python don't share a common makefile infrastructure. H
 * `make test` to run all of the test cases.
 * `make coverage` to view the current unit test coverage.
 * `make verify` to run [flake8](http://flake8.pycqa.org/en/latest/) over the code to detect errors.
+
+
+We do not use tools like pylint by default, as they can be too aggressive and provide a large number of benign errors, often in third party modules. However, it can be difficult to track down import errors using the standard test infrastructure we have in place, and it can be useful under these circumstances to be able to run pylint to draw out any bad imports that might be causing the modules or unit tests to fail.
```

Comment:
It would be useful to have an example of the import error you get (so that we can search on the error when we see it to find this page)